test/integration/targets/ansible-galaxy/aliases

```diff
@@ -1,2 +1,3 @@
 destructive
 shippable/posix/group3
+skip/python2.6  # build uses tarfile with features not available until 2.7
```

Comment:
I thought you wanted to target 2.6. What happened? Do we still need that CM then?

Reply:
The code requires the `filter` kwargs in `tarfile.add`. This is a lot more complex to try and support in 2.6 compared to the context manager work. Becasue of this we will have to skip the tests on 2.6.